lib/tools/xctest-commands.js

```diff
@@ -50,8 +60,27 @@ xctestCommands.installXCTestBundle = async function installXCTestBundle (xctestB
  * @returns {Array.<string>} the list of the xctest bundle ids
  */
 xctestCommands.listXCTestBundles = async function listXCTestBundles () {
-  const bundles = fixOutputToArray(await this.exec(['xctest', 'list', '--json']));
-  return bundles.map((bundle) => bundle.bundle_id);
+  const output = await this.exec(['xctest', 'list', '--json']);
+  try {
+    let bundles = fixOutputToArray(output);
+    bundles = _.isArray(bundles) ? bundles : [bundles];
+    return bundles.map((bundle) => bundle.bundle_id);
+  } catch (e) {
+    throw new Error(`Failed to parse '${output}' into json object: ${e.message}`);
```

Comment:
it makes sense to add an option to fixOutputToArray, so it does not mute JSON parsing exceptions